### PR TITLE
Update initdev

### PR DESCRIPTION
**Pull request by roylee17**

### CODE REVIEW

File: etc/initdev/Vagrantfile [hunk 1]
```diff
@@ -4,7 +4,7 @@
 VAGRANTFILE_API_VERSION = "2"
 
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
-  config.vm.box = "ubuntu/vivid64" # 15.04
+  config.vm.box = "bento/ubuntu-16.04"
```
Comment (jdoliner) [inline etc/initdev/Vagrantfile]: Is bento/ubuntu-16.04 a standard place to get an ubuntu vm from?

Comment (roylee17) [inline etc/initdev/Vagrantfile]: It seems to be the preferred place maintained by open sources. 

Refer to the last comment from @sethvargo in https://github.com/mitchellh/vagrant/issues/7155

Comment (jdoliner) [inline etc/initdev/Vagrantfile]: Good enough for me, I don't really use vagrant for dev.

**Comment by jdoliner** [comment]
This LGTM, is it ready for merge or is there more to come?

Also we'll need you to sign our CLA before we can merge: https://pachyderm.wufoo.com/forms/pachyderm-contributor-license-agreement/

File: etc/initdev/Vagrantfile (hunk 1 above)
Comment (roylee17) [inline etc/initdev/Vagrantfile]: Me, either. I thought it was the dev environment this morning, but then figured out the docker flow later on the slack. Will try the host environment later.

Comment (jdoliner) [inline etc/initdev/Vagrantfile]: Cool. What was it that lead you to the vagrant environment? We should probably correct that to nudge people toward the docker flow. Or was it just the existence of the vagrant file?

Comment (roylee17) [inline etc/initdev/Vagrantfile]: I guest it's just the existence. Tried host and docker first without luck in the first time. So turned to Vagrant, and realized it was just empty ubuntu + go.


Adding a dev guide section in the documentation will definitely help.
 
Another thing might help is to break up the Makefile a little, as it now has more than 60+ targets. Or at least reorder the targets so recommended flow will be spotted more easily.

For example, this morning I tried the following
`
make
make build
make deps; make build
make proto
`

Then I saw the Dockerfile(s) in the root. So
`
docker build -t pachyderm
docker run -it pachyderm
make deps; make build;
...
`
Still no luck. So I turned to slack for help.

However, if I just keep scrolling down the Makefile a coupe of pages, I might have found the docker-build and found it earlier.

**Comment by roylee17** [comment]
Okay. Signed the CLA.

**Comment by jdoliner** [comment]
Merged, many thanks :)

**Pull request merged**